src/components/modal/EditModal.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react";
+import "../../pages/home/home.css";
+import { Editor } from "../editor/Editor";
+import parse from "html-react-parser";
+import {
+  EditorWrapper,
+  TitleBox,
+  ButtonToNote,
+  CloseButton,
+  EditorFooter,
+  CheckBoxInput,
+  Checkboxlabel,
+} from "../../pages/home/homeComponents";
+
+import {
+  handleToggleEditModal,
+  editNoteHandler
+} from "../../Redux/Reducers/notesSlice";
+import { useDispatch, useSelector } from "react-redux";
+import { AlertToast } from "../toasts";
+import { ColorPallete } from "../pallete/ColorPallete";
+
+export const EditModal = () => {
+  const { noteToEdit } = useSelector((store) => store.notes);
+
+  const dispatch = useDispatch();
+
+  const [bgColor, setbgColor] = useState(noteToEdit.bgcolor);
+  const [note, setNote] = useState({
+    title: noteToEdit.title,
+    content: noteToEdit.content,
+    priority: noteToEdit.priority,
+  });
+  const [tags, setTags] = useState(noteToEdit.tags);
+  const [enable, setEnable] = useState(true);
+
+  const changeHandler = (e) => {
+    setNote((prev) => ({ ...prev, [e.target.name]: e.target.value }));
+  };
+
+  const setTagsHandler = (e) => {
+    const include = e.target.checked;
+    const value = e.target.value;
+    if (include) {
+      setTags([...tags, value]);
+    } else {
+      setTags([...tags.filter((tag) => tag != value)]);
+    }
+  };
+
+  const clickHandler = () => {
+    if (note.title === "") {
+      AlertToast("Please Enter Note Title");
+      return;
+    }
+    if (note.content === "") {
+      AlertToast("Please Enter note text");
+      return;
+    }
+    setEnable(false);
+    const parsedData = parse(`${note.content}`).props.children;
+    dispatch(
+        editNoteHandler({
+        _id:noteToEdit._id,
+        title: note.title,
+        content: parsedData,
+        bgcolor: bgColor,
+        priority: note.priority,
+        tags: tags,
+      })
+    );
+    dispatch(handleToggleEditModal());
+  };
+
+  return (
+    <>
+      <EditorWrapper style={{ backgroundColor: bgColor }}>
+        <CloseButton onClick={() => dispatch(handleToggleEditModal())}>
+          {" "}
+          X
+        </CloseButton>
+
+        <TitleBox
+          type="text"
+          placeholder="Add title ....."
+          style={{ backgroundColor: bgColor }}
```

Comment:
Try to avoid inline styling.

Reply:
Here I am setting bbackground according to the bgColor state that's why